src/main/java/com/timekeeper/app/dto/Payroll.java

```diff
@@ -12,14 +15,46 @@
     private double hoursWorked;
     private double amountPaid;
 
+    /**
+     * Sets the hours worked while making sure that a negative value cannot be set
+     * @param hoursWorked A double that reflects the amount of hours worked
+     */
+
+
+    public void setHoursWorked(double hoursWorked){
+        if (hoursWorked <0.0){
+            hoursWorked = 0.0;
+        }
+        this.hoursWorked = hoursWorked;
+    }
+
+    /**
+     * Returns the inputted double rounded to the inputted amount of places.
+     *
+     * @param value the double to be rounded
+     * @param places the number of decimal places to be rounded to
+     * @return an appropriately rounded double
+     */
+    public static double round(double value, int places) {
+        if (places < 0) throw new IllegalArgumentException();
+
+        BigDecimal bd = BigDecimal.valueOf(value);
+        bd = bd.setScale(places, RoundingMode.HALF_UP);
+        return bd.doubleValue();
+    }
```

Comment:
I'd make one more (very important) change.  As long as you're using a BigDecimal, use it as the return type as well.  doubles are subject to Floating Point Arithmetic, which can lead to inaccuracies.  BigDecimal is not.  So, as long as you're using BigDecimal, use it universally.